packages/react-components/react-avatar/src/components/AvatarGroup/renderAvatarGroup.tsx

```diff
@@ -1,13 +1,36 @@
 import * as React from 'react';
+import { Popover, PopoverTrigger } from '@fluentui/react-popover';
+import { Tooltip } from '@fluentui/react-tooltip';
 import { getSlots } from '@fluentui/react-utilities';
+import { AvatarGroupContext } from '../../contexts/AvatarGroupContext';
 import type { AvatarGroupState, AvatarGroupSlots } from './AvatarGroup.types';
 
 /**
  * Render the final JSX of AvatarGroup
  */
 export const renderAvatarGroup_unstable = (state: AvatarGroupState) => {
   const { slots, slotProps } = getSlots<AvatarGroupSlots>(state);
+  const { layout, size } = state;
 
-  // TODO Add additional slots in the appropriate place
-  return <slots.root {...slotProps.root} />;
+  return (
+    <AvatarGroupContext.Provider value={{ layout, size }}>
+      <slots.root {...slotProps.root}>
+        {state.root.children}
+        {state.hasOverflow && (
+          <Popover trapFocus size="small">
```

Comment:
We might also need a slot for this `Popover` so it can be configured by the user (fine to do in a separate PR).

Reply:
I added it to the AvatarGroup Epic, currently it's not possible due to the typings complaining about `Slot<typeof Tooltip>`

Reply:
Since `Popover` is not rendered as an element, so it cannot be made into a slot.